.zenodo.json

```
@@ -0,0 +1,162 @@
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Jonathan Gregory",
+      "affiliation": "University of Reading and UK Met Office Hadley Centre",
+      "orcid": "0000-0003-1296-8644"
+    },
+    {
+      "name": "Bob Drach",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Karl Taylor",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Steve Hankin",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Jon Blower",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "John Caron",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Rich Signell",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Phil Bentley",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Greg Rappa",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Heinke Höck",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Alison Pamment",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Martin Juckes",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Martin Raspaud",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Randy Horne",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Timothy Whiteaker",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "David Blodgett",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Charlie Zender",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Daniel Lee",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "David Hassell",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Alan D. Snow",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Tobias Kölling",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Dave Allured",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Aleksandar Jelenak",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Anders Meier Soerensen",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Lucile Gaultier",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Sylvain Herlédan",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Fernando Manzano",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Lars Bärring",
+      "affiliation": "SMHI Rossby Centre, Swedish Meteorological and Hydrological Institute",
+      "orcid": "0000-0001-7280-2502"
+    },
+    {
+      "name": "Christopher Barker",
+      "affiliation": "",
+      "orcid": ""
+    },
+    {
+      "name": "Sadie Bartholomew",
+      "affiliation": "",
+      "orcid": ""
```

Hi all (@castelao, governance members), I had a skim read of this PR since it came up on my notifications and I thought whilst I was here I might as well update my own details. If you accept the suggestion here they can go into this PR. (If for some reason you don't want this yet then they can be added later, but these are the relevant values for the fields in my case.)

```suggestion
      "name": "Sadie L. Bartholomew",
      "affiliation": "National Centre for Atmospheric Science and University of Reading",
      "orcid": "0000-0002-6180-3603"
```

---

Yes, I believe this is what we want to get for all the authors. Thanks!